src/hooks/usePantry.ts

```diff
@@ -161,6 +171,43 @@ export default function usePantry() {
     }
   }
 
+  enum Scopes {
+    DisplayName, Provides, ProjectName
+  }
+
+  async function find(input: string | { project: string }, scopes: Scopes[] = []): Promise<LsEntry | undefined> {
```

Comment:
What is the default? it is not clear from the definition or documentation

Reply:
moved to the definition